torchvision/models/detection/transform.py

```diff
@@ -18,7 +18,7 @@ def _get_shape_onnx(image: Tensor) -> Tensor:
 @torch.jit.unused
 def _fake_cast_onnx(v: Tensor) -> float:
     # ONNX requires a tensor but here we fake its type for JIT.
-    return v
+    return v  # type: ignore
```

Comment:
What is the error? Can we use
```suggestion
    return cast(float, v)
```

Reply:
No, this is needs to be as-is. :( 

It's a nasty workaround to deal with the fact that ONNX requires the value to be a Tensor (so that it's part of its computational graph). If you cast it, it will no longer work...

Reply:
Are you sure? `from typing import cast` is a no-op at runtime. It only exists to help `mypy`.

Reply:
Sounds good then, give it a try and if the test on ONNX does not fail then we can definitely keep it. Perhaps even edit the comment and explain that the cast is no-op at runtime and it's there only to help `mypy`.

Reply:
Seems that cast() is working fine for ONNX,